integration-tests/tests/mocked/nino-issue-credential/nino-issue-credential-happy.test.ts

```diff
@@ -1,7 +1,7 @@
 import { HistoryEvent } from "@aws-sdk/client-sfn";
 import { SfnContainerHelper } from "./sfn-container-helper";
 
-jest.setTimeout(30_000);
+jest.setTimeout(60_000);
```

Comment:
How come we've had to double the time out?

Reply:
I think these test started with 5_000 we've been doubling this, whenever we encounter an issue where we run these test locally and they pass, but then fail when github action runs them. Mainly it is something to do with do initialisation we have this ticket https://govukverify.atlassian.net/browse/OJ-2188

Reply:
Cool great to see the investigation captured in a ticket will resolve the comment 👍